src/SprykerEco/Yves/AmazonPay/Theme/default/components/atoms/confirmation-button/confirmation-button.ts

```diff
@@ -0,0 +1,81 @@
+import Component from 'ShopUi/models/component';
+
+declare const window: {
+    OffAmazonPayments: {
+        initConfirmationFlow(sellerId: string, orderId: string, callback: (confirmationFlow: any) => void): void
+    }
+};
+
+
+export default class ConfirmationButton extends Component {
```

Comment:
It's a bad name for this component. Why the confirmation button aware about xhr?

Reply:
fixed